### ISSUE TITLE
No indication a warning is inconclusive

### ISSUE THREAD

**Issue by firewave** [johnthagen/clion-cppcheck#41]
### Environment

* IDE Version (e.g. CLion 2016.3.2): CLion 2020.3 EAP
* Cppcheck executable version (`cppcheck --version`): 2.1
* Cppcheck plugin version: 1.4.2
* Exact strings used in cppcheck plugin options:
  * **cppcheck options**: --enable=all --inconclusive

### Expected behaviour

Warnings tooltips should indicate they are inconclusive.

I ran into analyzing a suspected false negative which turned out to be a inconclusive warning. I could confused since one nstallation showed the warning and one didn't. It turned out one was configured with `--inconclusive`.

### Actual behaviour

![image](https://user-images.githubusercontent.com/242857/95079729-51073080-0717-11eb-89e4-fd484e1bea4f.png)

```
test.cpp:14:22: warning:inconclusive: Access of moved variable 'a'. [accessMoved]
  std::string name = a.s_;
                     ^
test.cpp:13:8: note: Calling std::move(a)
  func(std::move(a));
       ^
test.cpp:14:22: note: $symbol:a
Access of moved variable '$symbol'.
  std::string name = a.s_;
                     ^
```

**Comment by johnthagen** [comment]
I seem to recall in IntelliJ there are different kinds of warning levels. Perhaps we could use "weak warning" or something similar to categorize inconclusive warnings.

**Comment by firewave** [comment]
I don't think they should have a different level since inconclusive could be warnings or errors. Just having it visible in the tooltip might be enough I think.

**Comment by firewave** [comment]
While at it... I also think it is not necessary to show the Cppcheck severity since that is already reflected by the CLion severity with the new `Problems` view that feel quite redundant:

![image](https://user-images.githubusercontent.com/242857/95108237-4d85a080-073b-11eb-960f-bb5b13ec2d66.png)

Adding the "inconclusive" in braces instead would make sense though.
